src/stk/databases/mongo_db/value.py

```diff
@@ -98,21 +101,49 @@ def __init__(
             key, they will return the same value from the database.
 
         lru_cache_size : :class:`int`, optional
+            This argument is deprecated and will be removed in any
+            version of :mod:`stk` released on, or after, 01/01/21.
+            Use the `put_lru_cache_size` and `get_lru_cache_size`
+            arguments instead.
+
             A RAM-based least recently used cache is used to avoid
             reading and writing to the database repeatedly. This sets
             the number of values which fit into the LRU cache. If
             ``None``, the cache size will be unlimited.
 
+        put_lru_cache_size : :class:`int`, optional
+            A RAM-based least recently used cache is used to avoid
```

Comment:
I do not understand 'A RAM-based least'. What does least mean here?

Reply:
it should be read as 'A RAM-based least-recently-used-cache'